docs/security-and-authentication.rst

```diff
@@ -34,3 +34,15 @@ local Runhouse config (:code:`~/.rh/config.yaml`), or in Python:
 
     import runhouse as rh
     rh.configs.disable_data_collection()
+
+
+Disabling Cluster Observability
+---------------------------------------
+To disable cluster observability, set the environment variable :code:`enable_observability`
+to :code:`False`. Alternatively, set :code:`enable_observability` to :code:`false` in your
+local Runhouse config (:code:`~/.rh/config.yaml`), or in Python:
```

Comment:
maybe we make this clear that doing so will disable telemetry across all subsequent clusters? since this is more of a "global" setting

Reply:
Add more information about what is a cluster observability, before explaining how to disable it.